### PR TITLE
Feature: action for publishing package update

### PR DESCRIPTION
**Pull request by talyguryn**
## Release publishing

There is an [action](.github/workflows/publish.yml) that fired on a new release publishing on GitHub.

After update merging, when a new package version is ready to be published,
create a [new release](https://github.com/codex-team/editor.js/releases/new) with the correct version tag.

Use target version changelog as a description.

Then you can publish the release and wait for package publishing via action.

This package version will be published to NPM with default `latest` tag.

### Release candidate publishing

If you want to publish release candidate version, use suffix `-rc.*` for package version in package.json file and git tag.

This package version will be published to NPM with `next` tag.

Stable version: `2.19.0`
Release candidate: `2.19.1-rc.0`, `2.19.1-rc.1`, ...
Next version: `2.19.1`

Do not forget to mark this release as a pre-release!

### CODE REVIEW

File: docs/releases.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+Do not forget to mark this release as a pre-release!
+
+![](https://capella.pics/796de9eb-bbe0-485c-bc8f-9a4cb76641b7.jpg)
+
+## Example pipeline
+
+Let's imagine that package version is `2.19.0` and you want to add some bug fixes and publish an update as `2.19.1`.
+
+1. Merge a single update or a few pulls with fixes to the main branch `next`.
```
Comment (neSpecc) [inline docs/releases.md:60]:
```suggestion
1. Merge a single update or a few pulls with fixes to the default branch `next`.
```

File: docs/releases.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+2. Bump the version up to `2.19.1-rc.0`. For the rest rc updates you should bump version number in suffix (to `2.19.1-rc.1` etc).
```
Comment (neSpecc) [inline docs/releases.md:61]:
```suggestion
2. Bump the version up to `2.19.1-rc.0` in the package.json. For the rest rc updates you should bump version number in suffix (to `2.19.1-rc.1` etc).
```

File: docs/releases.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+3. Create a new release on the releases page with tag `v2.19.1-rc.0` and mark "This is pre-release" checkbox.
+[Action](.github/workflows/publish.yml) will automatically push the package to NPM with tag `next`.
+4. When you ready to publish a release, remove suffix from version name, push changes and create a new release with tag `v2.19.1`.
+Same action will publish a new package as `latest` update.
```
Comment (neSpecc) [inline docs/releases.md:65]: Does it means that the `latest` package will be published before merging to `next` to the `master`?

File: docs/releases.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+![](https://capella.pics/57267bab-f2f0-411b-a9d1-69abee6abab5.jpg)
+
+Then you can publish the release and wait for package publishing via action.
+
+This package version will be published to NPM with default `latest` tag.
+
+### Release candidate publishing
+
+If you want to publish release candidate version, use suffix `-rc.*` for package version in package.json file and git tag.
```
Comment (neSpecc) [inline docs/releases.md:44]: > and git tag.

Can't find this step in the example pipeline

**Review by neSpecc — APPROVED** [review]

**Review by gohabereg — APPROVED** [review]

**Review by khaydarov — APPROVED** [review]

**Pull request merged**